modules/custom/az_person/config/install/views.view.az_person.yml

```diff
@@ -1494,6 +1494,890 @@ display:
+            make_link: false
+            path: ''
+            absolute: false
+            external: false
+            replace_spaces: false
+            path_case: none
+            trim_whitespace: false
+            alt: ''
+            rel: ''
+            link_class: ''
+            prefix: ''
+            suffix: ''
+            target: ''
+            nl2br: false
+            max_length: 0
+            word_boundary: true
+            ellipsis: true
+            more_link: false
+            more_link_text: ''
+            more_link_path: ''
+            strip_tags: false
+            trim: false
+            preserve_tags: ''
+            html: false
+          element_type: ''
+          element_class: ''
+          element_label_type: ''
+          element_label_class: ''
+          element_label_colon: false
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: false
+      sorts:
+        weight:
+          id: weight
+          table: draggableviews_structure
+          field: weight
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: ''
+            field_identifier: weight
+          exposed: false
+          draggable_views_reference: 'az_reorder:reorder_people'
+          draggable_views_null_order: after
+          draggable_views_pass_arguments: 0
+        field_az_lname_value:
+          id: field_az_lname_value
+          table: node__field_az_lname
+          field: field_az_lname_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: ''
+            field_identifier: field_az_lname_value
+          exposed: false
+        field_az_fname_value:
+          id: field_az_fname_value
+          table: node__field_az_fname
+          field: field_az_fname_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: ''
+            field_identifier: field_az_fname_value
+          exposed: false
+      style:
+        type: views_bootstrap_grid
+        options:
+          grouping: {  }
+          row_class: ''
+          default_row_class: false
+          col_xs: col-12
+          col_sm: col-sm-6
+          col_md: col-md-6
```

Comment:
col_md could be left at col-md-4 like on grid and grid modal. Those displays have determined that md viewport can support 3 cols, so there's no need to go down to 2 cols here.

Reply:
This would be more used in a case with a sidebar. Looking at that use case, the `col-md-6` makes more sense to me to not have content be so squished.

![image](https://github.com/az-digital/az_quickstart/assets/59745109/b9c9ac00-54f2-4564-a6ad-76eca638de61)

Reply:
I think it may be your own resolution that squishes it that much. Here's how it looks on my end using 1920x1080 on a couple PCs:
![image](https://github.com/az-digital/az_quickstart/assets/4400438/353d03e6-4407-4ce8-8941-a0d8ae19fd32)

As you can see, it looks like it has enough space with 3 columns and the sidebar.

Reply:
Follow-up note:
If you're meaning specifically the md range of 768-991px and not just the directory with sidebar in general, then I can see 2col being better there. But it also means the default assumption will be that sidebar will be used.

In that similar fashion, a separate issue should be created to change Grid Page and Grid Modal to use the same assumptions and set those to use 2col at md widths.